src/storage/storage-async.ts

```diff
@@ -49,15 +52,16 @@ import { Logger } from '../util/log';
 let logger = new Logger('storage async', 'yellowBright');
 let loggerSet = new Logger('storage async set', 'yellowBright');
 let loggerIngest = new Logger('storage async ingest', 'yellowBright');
+let J = JSON.stringify;
 
 //================================================================================
 
-export let docCompareForOverwrite = (newDoc: Doc, oldDoc: Doc): Cmp => {
-    // A doc can overwrite another doc if the timestamp is higher, or
-    // if the timestamp is tied, if the signature is higher.
+let docCompareNewestFirst = (a: Doc, b: Doc): Cmp => {
+    // Sorts by timestamp DESC (newest fist) and breaks ties using the signature ASC.
     return compareArrays(
-        [newDoc.timestamp, newDoc.signature],
-        [oldDoc.timestamp, oldDoc.signature],
+        [a.timestamp, a.signature],
+        [b.timestamp, a.signature],
+        ['DESC', 'ASC'],
```

Comment:
TODO: the earthstar spec doesn't say how to break timestamp ties between doc versions.  We need to formalize that.

Here I've changed it so the lowest signature wins, instead of highest like before, to match the rest of stone-soup.  I don't remember what classic Earthstar does.